apps/frontend/src/app/dashboard/dashboard.component.html

```diff
@@ -5,12 +5,13 @@ <h1>{{ 'DASHBOARD' | translate }}</h1>
     [rowspan]="mobileQuery.matches ? 1 : 2"
     [colspan]="mobileQuery.matches ? 9 : 6"
   >
-    <ccn-session-card style="width: 100%"></ccn-session-card>
+    <ccn-session-card
+      [refreshCurrentSession]="scanSubject"
+      style="width: 100%"
+    ></ccn-session-card>
```

Comment:
Es ist nicht gut, ein Subjekt als Input zu geben, es ergibt gar keinen Sinn.
Subjekt muss im Service definiert und in den Komponenten verwendet.
lass uns erst bitte reden, bevor du weiter daran arbeiten, weil wir ein paar Sachen ändern sollen